src/components/settings/SettingsSSOTab/SSOStepper.jsx

```diff
@@ -176,7 +181,7 @@ const SSOStepper = ({ enterpriseSlug, enterpriseId, enterpriseName }) => {
             )}
             <Stepper.ActionRow.Spacer />
             <Button
-              disabled={!isIdpStepComplete || idpNextButtonDisabled}
+              disabled={!currentError && (!isIdpStepComplete || idpNextButtonDisabled)}
```

Comment:
ooooh I like this! Good suggestion @kiram15

Reply:
I agree that what you have @marlonkeating is a sufficient fix here but if you have the energy, I'd definitely look into implementing an AlertModal

Reply:
I agree using an alert modal is a better error mechanism going forward.  However, using a modal will require it to be close-able and reopen-able, which is a combination I wasn't able to get working with the warning panel due to missing plumbing in the SSOConfigContext/SSOStepper complex.  

Could the missing plumbing be patched in?  Yes, but it would add yet more twists to a system that is already difficult for a newbie to wrap their heads around.